### DISCUSSION TITLE
immorality/caste-system/

### DISCUSSION THREAD

**Discussion by giscus** [SageofPHY6Paths/hinduismdebunked.com#5] [Comments]
# immorality/caste-system/

A website that debunks Hinduism

https://hinduismdebunked.com/immorality/caste-system/

**Comment by Its-alida** [comment]
Hey brother. I think you missed one whole skandh from Shrimad Bhagwatam, where non other than Shri NaradMuni himself says that " whoever does whatsoever activities , should be called of that varna " . He also says that one who does no wrong deeds , may have wrong habits only based on Kula Parampara. 

Sorry but your forefathers couldn't edit everything , I pity your efforts !!

**Reply by SageofPHY6Paths** [reply]
Sorry for late reply, I didn't see your post till now.

Can you mention exact canto and verse numbers from Srimad Bhagavatam or provide vedabase link like https://vedabase.io/en/library/sb/7/11/15/? I do remember somewhere seeing that varna is based on your habits and not based on birth in some scripture not sure if its in Bhagavatam. But 90% of the time when caste is mentioned in Hindu scriptures it is considered hereditary. Hinduism was written by a large number of people and is very inconsistent/hypocritical and I can understand that not everyone of them is an immoral casteist bigot. Some of the writers might not be casteist bigots.

I didn't understand your comment on forefathers. Are you trying to insult my ancestors because they were low caste Hindu shudras? Anyway I don't care if you insult my ancestors. I think they were very stupid fools, if you go back few centuries even people like Newton (his belief in alchemy) were very stupid. Perhaps because most Hindus have some **ancestor superiority complex** where they think their ancestors had nuclear bombs etc, you though insulting my ancestors might hurt me. But as an ExHindu, I don't have **ancestor superiority complex** like most Hindus.

**Comment by YashuuBatman** [comment]
Debunked all of this on 
"VedicVeracity" - TikTok 😁

**Reply by SageofPHY6Paths** [reply]
Your link: https://www.tiktok.com/@vedicveracity/photo/7373717353101561131

First verse in the video: Instead of looking at [Matsya Purana Part II CCXXVII.131](https://hinduismdebunked.com/immorality/caste-system/#part-ii-ccxxvii131) you saw XXVII.001. Similarly, for all other verses you just opened a wrong page and just said it is different. I know very well that religious people do not have enough reading capacity to find the verses so I even took a screenshot images like [matsya-purana[227.131].png](https://hinduismdebunked.com/assets/images/matsya-purana[227.131].png).

**Reply by ghost** [reply]
How tf do you idiots have TikTok in india. didn't your modi guy ban it

**Comment by ghost** [comment]
L take @SageofPHY6Paths

**Reply by ghost** [reply]
Hinduism solos

**Comment by Zneuromancer3** [comment]
Some of these translations are flawed. Brahmins can eat a food given by sudra, but since they usually eat non-veg that is why they are denied. I won't baselessly say "hinduism solos". I too am researching about a religion I love and will continue to worship. The only reason i am here because I don't want to simply say "Hinduism rocks" without learning both good and bad side. But i still love being a hindu

**Reply by Zneuromancer3** [reply]
also did you know for a fact that why krishna says not doing your duty is a sin? For example: when we are a student and don't study thats a sin. But lets say i finish school and get a job. Who will expect you to study then? Nobody expects you to pick up your 10th math textbook when you have a job right? So, getting a job is like a whole different caste than being a student. At that point you do the job you are assigned. Plus did you know for a fact that most humans born in kali yuga are sudra? SB 12.3.25. We are indeed crazy and out of our minds. Being all hate instead of respecting others opinions we get offended just because others don't accept us. Such a shame.

**Comment by VishakhaduttNew** [comment]
Hello, 

I am a practicing Hindu and would like to congratulate you on the efforts you have taken to put all the quotes on caste from various sources considered holy and authoritative by many Hindus. 

However, I wish to assert that the interpretations of Vedic verses are different from the translations or transliterations and should not be confused.

To study the Vedas one needs several reference points called 'vedangas.' These are phonetics (Shiksha), grammar (Vyakarana), etymology (Nirukta), metre or poetic structure (Chanda), ritual obscience (Kalpa) and temporal projections (Jyotisha).

Using the etymology of Rishi Yasya, Purusha in the Vedas would refer to the universal consciousness: "There is nothing unrelated, nothing isolated in the world. The universe is Purusha, a living, breathing, intelligent, self-organizing, sovereign system. " आत्म, आत्मा, वैश्विक तत्व, अमूर्त सार, शाश्वत, अविनाशी, निराकार, सर्वव्यापी. 
(An English translation of the Nirukta can be found here https://archive.org/details/nighantuniruktao00yaskuoft/mode/2up ) 

Moreover if you look at the metre and the other structural elements of this verse, you have quoted Rig Veda
RV 10.90.11: There are several sources that affirm that it is a later addition to the Rigveda (probably inserted later to legitimize the Varna system). Henry Thomas Colebrooke states that "that remarkable hymn (the Purusha Sukta) is in language, meter, and style very different from the rest of the prayers with which it is associated. It has a decidedly more modern tone, and must have been composed after the Sanskrit language had been refined." (See page 309 of https://archive.org/details/miscellaneouses00unkngoog/page/n324/mode/2up?view=theater)

Moreover, the Nirukta (etymology) itself is referred to as the ear of the Vedas—thereby connoting that references to body parts are to be taken as functional metaphors rather than literal (and therefore derogatory) meanings.

All this argument would not deny the fact that a caste system exists in Hindu society. But I would want to emphasize to you the difference between varna and jati and how the functions that a person performed in society acquired new meanings in economic and hereditary contexts and thereby created classes, hierarchies, and endogamy that persist today.

There is a huge difference between scripture and practice, and various families within the same geographical neighborhood continue to practice religion differently, although they are all classified under the amalgamous term Hinduism. 

As a true atheist, I expect you to be like the Charvaka school, structured in your arguments and dismissive of concepts through logical reasoning.

There have been serious critical analysis of Hindu thought over the centuries (of which the Charvaka school is an example). These have followed six methods (called pramanas = proofs) 
They are perception (pratyakṣa), inference (anumāna), testimony or authority (śabda), comparison and analogy (upamāna); postulation or derivation from circumstances (arthāpatti); and non-perception, or proof from absence (anupalabdhi). These are valid argumentative methodologies in Indian philosophy. For example, the Charvaka school upholds only perception as the authoritative source of knowledge and rejects the Vedas. Buddhism upholds both perception and inference and rejects śabda (or the authority of the Vedas as the epistemological source). Jainism holds śabda as valid but also accepts other authoritative sources, while Mimansa and Advaita Vedanta schools accept all six proofs as valid sources of knowledge.

What you are doing here is merely listing some scriptural quotations (not even in their canonical order and sometimes leaving gaps, such as no material under Ramayana and the Upanishads) and listing your collection of two interpretations of that quotation with your own witty annotations or interpolations, which neither recognize the immense time lapse between the writing of these scriptures and therefore their changing contextual interpretation nor explain their relevance to the current context where the majority of practicing Hindus may not have even heard of some of these scriptures. These cannot be accepted as serious criticisms and even fall short of debunking a religion that does not have a monolithic structure of adherence but is rather a vast, amorphous amalgamation of many belief systems.

I look forward to structured analytical arguments.

**Reply by SageofPHY6Paths** [reply]
>no material under Ramayana and the Upanishads

I was busy and haven't updated for a long time. I will soon read more scriptures and get back to adding stuff.

> There is a huge difference between scripture and practice, and various families within the same geographical neighborhood continue to practice religion differently, although they are all classified under the amalgamous term Hinduism.

Yes. All religions are vague. That's how they survive. Science makes precise predictions, like in particle physics, etc, and if they match with agreements, the theory succeeds; if not, the theory is thrown into the dustbin. All religions are vague, but Hinduism is much vaguer than any other religion. In every district you go to, the beliefs change, and there will be some local gods that no one outside that district knows. Due to the vagueness, people have no choice but to keep changing interpretations based on time and place.

>Using the etymology of Rishi Yasya, Purusha in the Vedas would refer to the universal consciousness

Ok. These more philosophical things about consciousness, epistemology, etc, came with [Upanishads](https://en.wikipedia.org/wiki/Upanishads), which are post-Vedic texts. If you retcon original Vedas and interpret Vedas using Upanishads, that will be literally the art of misinterpretation. All religions misinterpret embarrassing things. It is highly dogmatic if you change the meaning as much as necessary to make it look good.

>it is a later addition to the Rigveda (probably inserted later to legitimise the Varna system).

If you start saying these are later additions, how can you even trust Hinduism? Anything can be a later addition. See [List of languages by first written account](https://en.wikipedia.org/wiki/List_of_languages_by_first_written_account). Vedas were orally transmitted from 1500 BC. But the oldest written Sanskrit is 1st century BC, and that is not even Vedas. India is one of the last places to get scripture. The oldest known manuscripts of Vedas date to 1040 AD, so 2500 years after they were created. So, for 2500 years, pretty much everything will change. Then you should consider most of the Vedas as unreliable modern creations instead of cherry-picking only those verses that you don't like.

>For example, the Charvaka school upholds only perception as the authoritative source of knowledge and rejects the Vedas

I think the problem with Hindus is that to cope with the inferiority complex from British colonisation, they are more and more believing the pathetic myth that ancient India was more advanced than the modern world in everything from science to philosophy. I don't need to follow the epistemology of Charvaka just because I am an atheist and just because it was the only atheist philosophy in ancient India. Charvaka had a proto-version of empiricism. [Hume](https://en.wikipedia.org/wiki/David_Hume) developed a much better version of empiricism. But [Kant](https://en.wikipedia.org/wiki/Immanuel_Kant) challenged Hume's empiricism with his version of rationalism. Both Hume and Kant were atheist philosophers. These are not the only options either; there are 100s of modern atheist epistemologies. I don't need to follow the Charvakas just because I am an Indian ExHindu.

**Reply by SageofPHY6Paths** [reply]
Also @VishakhaduttNew what do you think is the single best argument for why someone should believe in Hinduism?

**Comment by VishakhaduttNew** [comment]
Hello,

Several new and different points mentioned in your reply, which really opens up the scope of discussion.

I'd like to take up the last point first; "what do you think is the single best argument for why someone should believe in Hinduism?"

I would desist from giving an evangelist answer to this question because that isn't my purpose. However, I'd certainly list here that the single reason why I choose to remain within the Hindu fold is because it accomodates skepticism. Many of its philosophical texts (Prsna Upanishad, Katha Upanishad, Bhagvad Gita, Ashtavakra Gita) are in question and answer format where the questioner is presented as a skeotic who rejects the current understanding or presents a counter view and asks for clarification. Treatises such as Yoga Vasistha, Yoga Yagnavalka, or Manisha Panchakam are also based on dialogue that starts on a skeptical note). In many such scenarios the initial state of the skeptic or the dissenter is expressed such as the Vishadyoga (dejection) of Arjuna that is the basis of the Bhagvad Gita or the Vairagya stithi (dispassion) of Sri Rama that is the start of the conversation with Rishi Vasistha in Yoga Vasistha or the frustration over meaningless rituals felt by Nachiketa that is the basis of the Kathopanishad. 

So the single reason I choose to remain within this fold is because of its accomodation to skepticism.

About Hinduism being vague, I'd like to use the word amalgamation because it signifies accomodation, which is what it really is. It allows so.many contradictory perspectives to coexist. This can be confusing, I agree, to the person who seeks consistency in direction but can also be liberating in a sense that it isn't restrictive.

But society and dogma persist and the existence of societal pressures and restrictive practices within almost all Hindu denominations cannot be denied. All that I am saying is that it is not part of the Vedas. 

That brings us to the question of later additions and the question of trust. Later additions are definitely being discovered in several texts throughout history (not only Hindu history). Even Christian Canonical works went through a weeding where apocryphal texts were identified and called out. This has been a theological as well as academic exercise across generations. Expounding and weeding occur in every belief system.

As far as the question of trust goes, you are right and spot on to call out the difference between religion and science when you say that science strives to be precise in its predictions. However Karl Popper, the foremost among philosopers of science, states that falsifiability is the cornerstone of science. No scientific thesis can claim to be absolute and if it asserts that it is so then it doesn't adhere to scientific principles. Religion on the other hand refers to shashvat or eternal truths that cannot be falsified. So basically all those who state that their religion is "a scientific religion" are actually stating that their religion can be falsified! 

Hindu religion isn't scientific simply because it is religion. It welcomes people who do not trust it ( at the very least, the Vedic treatises seem to accomodate that). Over the centuries it has branched out into several different sects and sub-sects each having their own rigid practices but it has also seen change and has transformed over time.

Cheery-picking verses that one likes or does not like - inherently means that you are trying to defend something through selective reading. I am not even attempting to do that. I am only commenting on a verse that you have picked and restricting myself to only that verse. Had I followed the principle of cherry picking, I could have countered the verses you picked with another verses that state the opposite ( there are several such verses in the Maha Upanishad, Kathopanishad, Skanda Purana, Manisha Panchakam and other texts that expound exactly the opposite view about caste system but ultimately both prove that the caste system existed and that therefore were views that justified and opposed it)

Coming to the point of considering the Vedas as unreliable because there was a significant oral tradition of transmission over centuries (from 1500 BCE to 1040 CE ) I disagree with that entirely because even Michael Witzel states that the unbroken line of oral transmission of the Vedas was so impeccable and superior to the classical (scripted) texts of other cultures that it was almost like a "tape recording" where not only the words but even the musical (tonal) accents have been preserved right upto the present. (Please refer to ISBN 1-4051-3251-5 ). Additionally as I mentioned earlier Shiksha (phonetics) Vyakrana (grammar) and Chanda (poetic metre) are essential to studying the Vedas in addition to Nirukta (etymology) Kalpa (ritual obsciences) and Jyotisha (temporal projections). So it is on the basis of these perspectives alone that Colebrooke and even Dr B.R Ambedkar have doubted the veracity of the Purusha sukta that you quoted. So to summarize - the Vedas are not unreliable but there are at the very least a few researchers that I can quote who have presented arguments that the verse that you have quoted seems to be a later addition.

Coming to your point regarding the belief that ancient India was more advanced in everything from science to philosophy - I believe that this belief was also   introduced by "orientalists" such as Madame Helena  Blavatsky, Nicolas Notovich and other people who were looking for some sort of  "esoteric" knowledge sources. Such people still exist today such as David Frawley, Francios Gautier and Eric Von Daniken who continue to feed some individuals who are looking for foreign endorsement of their alleged super history. However  their views cannot be taken to be representative of the a entire Hindu religion or a vantage point for criticising the Vedas or Hinduism per se. We all know and agree that such views are flights of fancy and cannot be credited  and serious thinkers do not subscribe to such views.

Lastly, I certainly do not want you to follow the epistemology of the Charvaka School. There are other aethistic  (निरीश्वरवाद) schools within Hinduism ( such as Sankhya and Mimansa) and other aethistic schools that do not accept the authority of the Vedas ( नास्तिक) such as अजिवका and अज्ञाना whose arguments you can study in addition to Kant and Hume however I was just suggesting a methodical and argumentative approach in place of a listing of annotated quotations.

I found Kant's 'Critique of Pure Reason' very illuminating and insightful.

**Reply by VishakhaduttNew** [reply]
A grammatical correction, My words "Hindu religion isn't scientific simply because it is religion."

Should be construed as "Hindu religion ISN'T scientific. Simply because it IS religion.

**Reply by SageofPHY6Paths** [reply]
You should have replied under [your previous comment](https://github.com/SageofPHY6Paths/hinduismdebunked.com/discussions/5#discussioncomment-12402783).

If you don't have an argument for why you believe in Hinduism, then you are blindly believing. Christians argue using the cosmological argument that you need a god and then saying many people saw the miracles of Jesus so he is the God. Their argument it bad but at least they are trying.

Also you are mistaken if you think Hinduism allows free speech and questioning. Kings that belived in Shavism used to kill anyone who worshipped Vishnu in ancient Tamil Nadu. So they kill even other Hindus and oppress them based on caste etc. For atheists etc they care much less about their rights. Check [SB 4.4.17](https://hinduismdebunked.com/docs/immorality/intolerance/#sb-4417) that says cut the tongue of atheists.

**Reply by Zneuromancer3** [reply]
It’s clear that you have a strong passion for analyzing Hinduism, but I feel like your approach is limited by an inherent bias. You seem more focused on “debunking” Hinduism rather than truly understanding it in its entirety. Questioning is good—it’s even encouraged in Hindu philosophy—but real questioning requires open-mindedness, not just looking for reasons to tear something down.

First, your argument that Hinduism allows no free speech or questioning is flawed. If that were true, how did schools like Charvaka (which rejected the Vedas), Sankhya, Mimamsa, and Ajivika thrive for centuries within India? Hindu texts themselves are filled with skepticism—look at the Bhagavad Gita, where Arjuna begins by questioning everything, or the Upanishads, which are structured as debates between teachers and students. You’re taking historical examples of persecution (which exist in every religion) and claiming that’s what Hinduism itself teaches. No, that’s just people using religion as a tool for control—something you should be fighting against rather than attacking those who are sincere in their devotion.

You referenced SB 4.4.17, but are you a scholar of Sanskrit? Relying on English translations alone can strip away deeper meanings. The British, in particular, mistranslated and misrepresented Hindu texts to weaken Indian self-identity, and many of these distortions remain today. If you’re truly dedicated to understanding, learn Sanskrit and read from original sources instead of just taking out-of-context translations. Even Dr. B.R. Ambedkar and scholars like Michael Witzel have pointed out that certain verses (like Purusha Sukta) were likely later additions. So cherry-picking verses to “debunk” Hinduism is just intellectual dishonesty.

Some people lose faith in God due to suffering. But karma isn’t some unfair system—people create their own circumstances through their continuous actions. Many want to do bad things but never face consequences. However, past-life karma is harder to grasp because we don’t remember it. But maybe that’s intentional. If we did, we might be crushed by guilt or overwhelmed by past emotions. Forgetting allows us to start fresh while still growing through life’s lessons.

Ultimately, you should ask yourself—what are you really trying to achieve? If you want to fight oppression and caste discrimination, I’m with you. But attacking those who sincerely follow a path of dharma, love, and self-improvement? That’s just misdirected energy. The problem is not Hinduism itself—it’s the people who twist it for power. Instead of blindly attacking, why not be part of a real solution?

**Reply by SageofPHY6Paths** [reply]
@Zneuromancer3 you are brainwashed at a young age to believe in religion. Few people like me see through the nonsense after they grow up. Unfortunately you do not have enough mental strength to see through the brainwashing.

https://vedabase.io/en/library/sb/4/4/17/ You ignorantly said it was British translation. No, the translation I used is by the guy who started ISKCON. He gave word by word meaning for it. I always give multiple translations because I know that religious people like to say the translation is incorrect whenever they see something they don't agree with. This is the art of misinterpretation.

>truly understanding it in its entirety

This is not science as Hinduism contains nothing to understand in it. It is just some fictional stories that are not even interesting compared to modern fictional stories that have more character depth etc.

>Ultimately, you should ask yourself—what are you really trying to achieve?

This might be hard to comprehend for a religious person like you. But I want to protect everyone from brainwashing. For that religions should go extinct. Hinduism has so many immoral things like caste discrimination, gods raping women, sati etc. But religions like Buddhism, Jainism don't have those evil things. ***But I want even those religions to go extinct***. Because even Buddhism & Jainism brainwash children with pseudoscience like atman, rebirth, karma, nirvana, etc. I don't think you are an idiot, if you were raised without teaching Hinduism until you became an adult you wouldn't believe in it.

>how did schools like Charvaka (which rejected the Vedas), Sankhya, Mimamsa, and Ajivika thrive for centuries within India? 

Who said they thrived? There are many scriptures that tell to persecute them. Maybe the guys who wrote them didn't follow them? No they followed, see https://wellburrowed.com/2019/10/15/lynching-in-ancient-india-the-case-of-charvaka/

@VishakhaduttNew

**Reply by Zneuromancer3** [reply]
Firstly, you didn't answer my question wether you were a scholar of Sanskrit. If you want honesty, then I am not a scholar of Sanskrit. But the Bhagavad Gita has been my guide and gives useful solutions to problems even that of today.

Secondly, even I don't trust the founder of ISKCON 

You assume that everyone who believes in religion is brainwashed, but that itself is a biased belief. If someone freely chooses faith based on their experiences, are they still brainwashed?

You claim to 'protect' people from brainwashing, yet you are pushing your own worldview just as dogmatically as religious extremists do. Atheism can also be dogmatic if it refuses to acknowledge anything beyond materialism.

I agree that many people have misused Hinduism for control—whether it’s caste discrimination or outdated practices. But the same is true for all ideologies, including atheism (look at the Soviet Union banning religion and persecuting believers).

You dismiss everything in Hinduism as 'fictional stories,' but that’s a shallow way to look at it. Stories are meant to convey wisdom, not just entertainment. Even modern fiction uses myths and archetypes. The Bhagavad Gita isn't just a story; it's a philosophical dialogue on duty, morality, and free will.

You say that if I wasn't raised with Hinduism, I wouldn't believe it. Maybe. But you also wouldn't believe in your current worldview if you weren’t raised in a secular or atheist-influenced environment. If logic alone could determine belief, then all intelligent people would be either theists or atheists—but even the greatest minds throughout history have disagreed.

Instead of focusing on destroying religion, why not focus on fighting the real issues—caste discrimination, religious oppression, and blind dogma? You don’t have to destroy spirituality to do that. Attack the people who corrupt religion, not the essence of faith itself.

Also, there are things that cannot be put into words or equations. Human emotions are complex. They are natural, but words and equations are man made. Some things are felt to be understood. Experiences that are beyond the limitations of the 5 senses that cannot merely be explained. If what you say is true then concepts like atma, nirvana and rebirth wouldn't exist in the first place.

You are assuming that one dramatic story in an epic defines all of Hinduism, but that’s a weak argument. If Hinduism wanted to erase materialist philosophy, why does it preserve records of Charvaka’s school and debates? Why do texts like the Upanishads encourage questioning?

Yes, some people in power have historically persecuted dissenters. But that’s not unique to Hinduism—it has happened in every civilization. The real question isn’t whether Hinduism has had dark moments, but whether its core philosophy supports free thought. And when you actually study it, you’ll find that it does.

Rather than cherry-picking extreme cases, let’s look at the full picture: Hinduism has room for atheism, skepticism, and devotion alike. The solution isn’t to destroy religion—it’s to remove the people who misuse it for control.

I still don't know why people think science is complete. There's still a whole universe worth of things to be discovered, to be understood. Science is incomplete, but i don't blindly trust religion. And i don't even like calling myself a Hindu. I'm a mere devotee of lord Krishna and nothing more. Science may have answers to the existence of God or might completely prove it as blasphemy. But until all of science is understood in God I will continue to believe.

I have had many of what you may call "coincidences" whenever I lose my belief in God. Like there is a possibility of his existence. But I don't think my personal experiences can be used as proof since it is all beyond current logic. Like saying the universe can "speak". All I say is attack those who use religion to control people, real conmen. Not those who want to follow devotion and dharma.

**Reply by SageofPHY6Paths** [reply]
> But you also wouldn't believe in your current worldview if you weren’t raised in a secular or atheist-influenced environment

I grew up drinking cow urine in a devout Hindu family. I wasn't raised in an atheist family.

>I still don't know why people think science is complete. 

Science is not complete. That is the beautiful thing about science. Because it is incomplete every day science improves. Every day some scientist finds something and it increases our knowledge of science even if its a small discovery. It is religion that claims completeness and **_never updates_** itself and keeps millennia old pseudoscientific dogma.

>Firstly, you didn't answer my question wether you were a scholar of Sanskrit

This isn't a new argument. Stupid Christians have been using this argument for many years. If someone believes in Christianity by reading an English translation they are fine with it. But if someone questions it they say you need to know Hebrew and the extinct Aramaic to read the original text before questioning. For believing, its fine if you know very little about these stupid books but to question them, I need to know every minor detail about these stupid books. What a stupidly unfair demand @Zneuromancer3.

**Reply by Zneuromancer3** [reply]
I can see that you’re set in your beliefs, just as I am in mine. But belief or disbelief shouldn’t blind us to the truth. My only point was that religion, like any system, is shaped by the people who follow it. The problem is never devotion or philosophy—it’s how people twist it for control. If you truly care about justice, fight against the people who exploit religion, not those who find peace in it.

Science is beautiful because it evolves, but it doesn’t yet have all the answers. Dismissing things just because they haven’t been 'proven' yet is as dogmatic as blind faith. Some experiences simply go beyond words or equations, and that’s something I’ve felt personally. If you haven’t, that’s fine. But I’m not here to prove anything to you—only to live by what I’ve seen and felt. In the end, truth prevails. Satyam eva jayate.

It’s not about knowing every minor detail, but understanding context. Shiva never ‘raped’ Mohini—Vishnu’s maya cannot be overpowered by Shiva. And even in other cases, Hinduism never justifies bad actions. It presents them as stories to show that evil never truly wins. Negative actions always have consequences, which is why you shouldn’t cherry-pick verses without providing full context. But I suppose that would spoil your Hindu debunking plans, wouldn’t it?

And about cow urine—yes, it does have scientifically proven medicinal properties. Just because something is mocked doesn’t mean it lacks value. The world isn’t as black and white as you make it out to be. @SageofPHY6Paths

**Comment by VishakhaduttNew** [comment]
Hello,

A lot seems to have been discussed over the weekend.

I will start with a word of caution.
The verse that you quoted from Shrimad Bhagvatam, 4.4.17 cannot be read in isloation. Even if one looks at the previous verse, 4.4.16, the context can be ascertained.

Shrimad Bhagvatam 4.4.16 states
किं वा शिवाख्यमशिवं न विदुस्त्वदन्ये ।
ब्रह्मादयस्तमवकीर्य जटाः श्मशाने ।
तन्माल्यभस्मनृकपाल्यवसत्पिशाचैः ।
ये मूर्धभिर्दधति तच्चरणावसृष्टम् ॥ १६ ॥

whether (किं वा) the name Shiva (शिवाख्य) seems inauspicious (अशिवं) learned personas other than you may not know (न विदुस्त्वदन्ये) of Him, Lord Brahma and others like him (ब्रह्मादयस्तम) though he is of scattered hair (वकीर्य जटाः ) and lives in crematoriums (श्मशाने) wears a necklace of skulls and is smeared with ashes (तन्माल्यभस्मनृकपाल्य) and associated with demons (सत्पिशाचैः) yet they (ये ) place over their head (मूर्धभिर्दधति) the flowers that fall from his lotus feet (तच्चरणावसृष्टम्) 

It is clear from this verse that it is Mata Sati talking to her father, Daksha (Who is a prajapati like Lord Brahma). She is criticising him for disrespecting her consort, Lord Shiva - whom even Brahma and the other prajapatis venerate. So it follows that in the next verse, SB 4.4.17, she says,

कर्णौ पिधाय निरयाद्यदकल्प ईशे ।
धर्मावितर्यसृणिभिर्नृभिरस्यमाने ।
छिन्द्यात्प्रसह्य रुशतीमसतीं प्रभुश्चेत् ।
जिह्वामसूनपि ततो विसृजेत्स धर्मः ॥ १७ ॥

One can block one's ears (कर्णौ पिधाय) and go away (निरयाद) or if the option is possible (अकल्प) or the enemy (सृणि) of the Lord (ईशे) who is the personification of Dharma (धर्मावितर्य)[ therefore enemy of the personified Dharma (धर्मावितर्यसृणि) ] Cut off (छिन्द्या) with all the strength you can muster (त्प्रसह्य) their speech (जिह्वा) that who fearlessly (नृभिर)  utters unpalatable disrespectful (अरस्यमाने) and disagreeable speech (रुशती) that is also impure (असतीं) and directed towards the Lord (प्रभुश्चेत् ) , and in countering in such manner (ततो) one should give up one's life (सूनपि) towards the cause of Dharma (विसृजेत्स धर्मः)

So a simple reading of these verses tells us about the state of Mata Sati's mind. She is so disturbed by the disrespect shown to her Lord, who is Dharma personified, that she can no longer just ignore (shut her ears and choose to walk away). She finds it honorable to resist such disrespectful, disagreeable, and unplatable speech by choosing to give up one's own life in protest. 

There are many examples of people who chose self-immolation as a form of protest. The most famous example I can cite here is of the monk, Thich Quang Duc, a Vietnamese Buddhist monk, who famously set himself on fire in protest against the persecution of Buddhists by the govt of South Vietnam in 1963.  

What you have done is
a) quoted without presenting the context
b) presented that this verse "says cut the tongue of atheists" when there is no mention of atheists, and
c) presented this verse as promoting violence when it talks of countering speech and not cutting off someone's (in this case her father's) tongue (which is unimaginable). 

Everyone familiar with Hinduism knows what happened in the story. Mata Sati chose to immolate herself in protest against the disrespect shown to her Lord who is Dharma personified. 

When such text is presented without context, it stands the risk of being cited by others who may use this mention as a reference for proving the promotion of violence against another person, which this text certainly doesn't promote.

**Reply by Zneuromancer3** [reply]
I agree with your statement. @SageofPHY6Paths  seems to take mistranslated verses or take them completely out of context. Also they seem to believe just because we follow a certain religion, we are brainwashed. I want them to debunk with context and proper translations. Otherwise they must attack only those who misuse religion for personal benefits and not attack those who have found meaning in it.

**Reply by VishakhaduttNew** [reply]
I agree with you that debunking concepts must happen within context and with correct translations. I am in favor of calling out those who misuse religion for personal benefit and respecting people who have found meaning in the religion of their choice. 

I also agree with you that reading as close to the original version as possible, as doing so will also bring out the beauty inherent in the original meaning.

A great example would be knowing that the name "Jesus" is the transliteration of the Hebrew name "Yeshua" (יֵשׁוּעָ)—a combination of "Ya" (an abbreviation for "Yahweh," the name of Israel's God) and the verb "yasha" (meaning "rescue," "deliver," or "save"). So "Jesus" means "Yahweh saves" or "Yahweh is salvation.". The angel Gabriel told Joseph in Matthew 1:21 to name his child "Jesus," signifying his role as savior. 

The word Christ comes from the Greek "Christos" (Χριστός), which translates to "anointed one" or "Messiah.". So simply knowing the meaning of "Jesus Christ" takes us closer to the core of Christian belief in such a beautiful way.
